src/modules/launcher/Wox.Test/Plugins/WindowsIndexerTest.cs

```diff
@@ -156,45 +156,6 @@ public void ExecuteQuery_ShouldDisposeAllConnections_AfterFunctionCall()
             Assert.IsTrue(oleDbSearch.HaveAllDisposableItemsBeenDisposed());
         }
 
-        [Test]
```

Comment:
Can you please add some test cases ? You can check if `QueryWhereRestrictions` contains correct `BITWISE` check based on `DisplayHiddenFiles` value.

Reply:
This is indead something that needs to be tested, i've added a true/false case.